src/main/java/com/englishtown/vertx/elasticsearch/SearchOptions.java

```diff
@@ -35,6 +35,9 @@
     private JsonObject aggregations;
     private List<SortOption> sorts = new ArrayList<>();
     private JsonObject extraSource;
+    private String templateName;
+    private String templateType;
```

Comment:
Mm, I didn't realized it worked out of the box with enums but I supposed I originally thought maybe we shouldn't let the ElasticSearch driver enum "bleed" to this service. Though since the request builder takes the enum directly rather than a string, it's probably safer to expose an enum.